### ISSUE TITLE
Virtual Service list view not working when there's an issue with a Virtual Service

### ISSUE THREAD

**Issue by djannot** [solo-io/gloo#3521]
**Describe the bug**
When there's an issue with a Virtual Service, you'll be able to see the Virtual Services in Grid View, but not in List View.
When click on the List View button, you get the following error:

```
There was an error with the Virtual Services Listing
```

**To Reproduce**
To reproduce the issue, you can simply add a route for an upstream that doesn't exist.

**Additional context**
Gloo version:

```
Client: {"version":"1.4.9"}
Server: {"type":"Gateway","enterprise":true,"kubernetes":{"containers":[{"Tag":"1.4.10","Name":"grpcserver-ui","Registry":"quay.io/solo-io"},{"Tag":"1.4.10","Name":"grpcserver-ee","Registry":"quay.io/solo-io"},{"Tag":"1.4.10","Name":"grpcserver-envoy","Registry":"quay.io/solo-io"},{"Tag":"1.4.11","Name":"discovery","Registry":"quay.io/solo-io"},{"Tag":"1.4.10","Name":"extauth-ee","Registry":"quay.io/solo-io"},{"Tag":"1.4.11","Name":"gateway","Registry":"quay.io/solo-io"},{"Tag":"1.4.10","Name":"gloo-ee-envoy-wrapper","Registry":"quay.io/solo-io"},{"Tag":"1.4.10","Name":"gloo-ee","Registry":"quay.io/solo-io"},{"Tag":"1.4.10","Name":"observability-ee","Registry":"quay.io/solo-io"},{"Tag":"1.4.10","Name":"rate-limit-ee","Registry":"quay.io/solo-io"},{"Tag":"5","Name":"redis","Registry":"docker.io"}],"namespace":"gloo-system"}}
```

**Comment by sirrapa** [comment]
@djannot,  @kdorosh , or @ascampos 

Wil this be backported to the 1.4.x branch?

**Comment by kdorosh** [comment]
we released v1.4.13 with the fix @sirrapa
